app/ceom/photos/templates/photos/leaderboard.html

```diff
@@ -0,0 +1,114 @@
+{% extends "photos/base.html" %}
+{% load i18n %}
+{% block content %}
+
+<h2 style="color: #0860be;" >Photos Leaderboard</h2>
+
+<h5 class="text-center"> Welcome to the <strong> CEOM Photos Leaderboard</strong>! </h5>
+
+<p class="text-center"> Ranks are calulated based on which user classifies the most photos.</p>
+    
+<!-- <div class="mt-3 d-flex justify-content-center"> 
+    <button type="button" class="btn btn-light btn-sm"> <a href="#position"> Jump To Your Position </a> </button>
+</div> -->
+
+<br>
+
+<table class="table table-bordered" style="width: 75%; border:1px solid black;margin-left:auto;margin-right:auto;">
+    <thead class = "table-dark">
+      <tr style="text-align: center">
+        <th scope="col">Ranking</th>
+        <th scope="col">Username</th>
+        <th scope="col">Points</th>
+      </tr>
+    </thead>
+
+    <tbody>
+      {% for users in page %}
+        {% with index=forloop.counter0 %}
+            <tr style="text-align: center">
+                {% ifequal users.username|stringformat:"s" request.user.username %}
```

Comment:
Interesting use of Django template language here for the string formatting, I haven't seen this before

Good job on the research